_component_design/toast.md

```diff
@@ -4,11 +4,11 @@ parent: Notifications
 layout: component
 category: Components
 usage: >
-  Toasts are a specialized notification component used to provide feedback on the status of an actions taking place in the background. Toasts are meant to be ephemeral and capture users attention by having the highest Z-index of all components.
+   A toast is a notification component that contains a short message about the status of a user action. Toasts appears on the screen for 10 seconds and then disappears.
```

Comment:
taking notes here: 
toast allows user to navigate away from a process while it happens and the user is notified in their current context when the process completes (or or fails).

the phrase "user initiated process" seems like a useful one here.

please add: 
do not use a toast for form validation.
do not use a toast if a users acknowledgement of the message is required .

Reply:
Done.